blueprints/ember-frost-theme/index.js

```diff
@@ -1,18 +0,0 @@
-module.exports = {
```

Comment:
I did not modify it when making the original change, so I left it as it was before.